release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -576,6 +576,35 @@ In addition to the features provided in the previous pre-releases, this version
 * Improvements to supportability, such as integration gathering and better status reporting.
 * A method for in-field emergency configuration overrides was devised and documented.
 
+[id="ocp-4-6-optimizing-data-plane-performance-with-intel-devices"]
+==== Optimizing data plane performance with Intel devices
+
+{product-title} {product-version} supports the:
+
+* OpenNESS Operator for Intel FPGA PAC N3000
+* OpenNESS SR-IOV Operator for Wireless FEC Accelerators
+
+[NOTE]
+====
+The N3000 Operator requires a prebuilt driver container with the Open Programmable Accelerator Engine (OPAE) out of tree kernel modules. This is built and shipped by Intel and currently supported with {product-title} 4.6.16.
+If a different {product-title} version is requested please contact Intel through their premier support portal at link:http://premiersupport.intel.com[Intel® Premier Support Access] or at mailto:openness.n3000.operator@intel.com[openness.n3000.operator@intel.com].
+
+For more details, see link:https://catalog.redhat.com/software/operators/detail/6001a748e4e3f23b0b6ad765[OpenNESS Operator for Wireless FEC Accelerators].
+====
+
+These Operators support the requirements of vRAN deployments for low power, cost and latency, whilst also delivering the capacity to manage spikes in performance for a range of use cases. One of the most compute-intensive 4G and 5G workloads is RAN layer 1 (L1) forward error correction (FEC), which resolves data transmission errors over unreliable or noisy communication channels.
```

Comment:
Please also apply whatever changes you make to https://github.com/openshift/openshift-docs/pull/33356 to this PR as well.